gobblin-core/src/main/java/gobblin/writer/AvroHdfsDataWriter.java

```diff
@@ -11,8 +11,14 @@
 
 package gobblin.writer;
 
+import com.google.common.base.Preconditions;
```

Comment:
Thanks. Out of curiosity, what happens if the import is not one of java, org, com, gobblin?